package.json

```diff
@@ -144,7 +144,7 @@
                     },
                     "vscode-vdf.popfile.format.tabs": {
                         "type": "number",
-                        "markdownDescription": "How many tabs to insert betweens keys and values when formatting. `-1` = Insert a single space",
+                        "markdownDescription": "How many tabs to insert between keys and values when formatting. `0` = Insert a single space",
                         "default": -1
```

Comment:
Should the default change to `0` with this also?

Reply:
Perhaps? I think it's about making the user understand it better, like with arrays, where the first position is 0

Reply:
Yes.